Taskfile.yaml

```
@@ -55,6 +55,27 @@ tasks:
+    env:
+      CGO_ENABLED: 0
+    cmd: |
+      go build -gcflags "all=-N -l" -o bin/reportportal-mcp-server-debug cmd/reportportal-mcp-server/main.go
+
+  debug:stop:
+    desc: "Stops any running debug processes on port {{.DLV_PORT}}"
+    cmd: powershell -Command '$conns = Get-NetTCPConnection -LocalPort {{.DLV_PORT}} -ErrorAction SilentlyContinue; if ($conns) { $conns | Select-Object -ExpandProperty OwningProcess -Unique | ForEach-Object { Stop-Process -Id $_ -Force -ErrorAction SilentlyContinue }; Write-Host "Stopped processes on port {{.DLV_PORT}}" } else { Write-Host "No process found on port {{.DLV_PORT}}" }'
+
+  debug:
+    desc: "Runs the MCP server locally in debug mode with Delve ({{.MCP_MODE}}). Debugger listens on port {{.DLV_PORT}}. Use DLV_PORT env var to change port."
+    deps: [ app:build-debug ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)